### PR TITLE
Adds a new meat steak sprite

### PR DESCRIPTION
**Pull request by Alberyk**
As suggested by this topic: https://forums.aurorastation.org/topic/12813-steak-resprite

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1701,7 +1701,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/meatsteak
 	name = "meat steak"
 	desc = "A piece of hot spicy meat."
-	icon_state = "meatstake"
+	icon_state = "stake"
```
Comment (fernerr) [inline code/modules/reagents/reagent_containers/food/snacks.dm]:
```suggestion
	icon_state = "steak"
```

**Comment by alsoandanswer** [comment]
hm, maybe you could implement changing the sprite when you bite into it

Like if reagents > reagent_number
icon state = bite-1
etc.

**Comment by Alberyk** [comment]
Well, the issue is that you could somehow regenerate the steak by injecting it with more reagents.

**Comment by alsoandanswer** [comment]
Regenerating steaks aren't *that* bad.

But if you are that particular, maybe use a different var for number of bites taken that only can go up instead of reagents or use a percentage based on reagents that can only go down.

**Comment by KingOfThePing** [comment]
fucking finally, this is the immersion I am looking for

**Comment by Alberyk** [comment]
This has been fixed, and it uses all icons for different states of the steak being eaten.

**Pull request merged**